### ISSUE TITLE
Authentication fails on 3.0.0

### ISSUE THREAD

**Issue by acid-chicken** [ShyykoSerhiy/vscode-spotify#51]
# Steps to reproduce

1. Push *Spotify Sign In* button.
2. Firefox opens <https://vscode-spotify-auth.azurewebsites.net/?message=You%20can%20now%20close%20this%20tab>.
3. vscode-spotify says *Failed to retrieve access token : "state_mismatch"*

![image](https://user-images.githubusercontent.com/20679825/46571405-80132c00-c9af-11e8-84a9-2d3ddf4d2b8f.png)

https://github.com/ShyykoSerhiy/vscode-spotify/blob/e753c11bb0261d84a1455f1984744f7d39e6e37d/src/actions/actions.ts#L234

# Environments

<dl>
<dt>vscode-spotify version</dt>
<dd>3.0.0</dd>
</dl>
<dl>
<dt>vscode version</dt>
<dd>1.27.2</dd>
</dl>
<dl>
<dt>os</dt>
<dd>Linux 4.18.5-arch1-1-ARCH #<span/>1 SMP PREEMPT Fri Aug 24 12:48:58 UTC 2018 x86_64 GNU/Linux
</dd>
</dl>

**Comment by levrik** [comment]
@acid-chicken Maybe try to remove access for vscode-spotify and try the whole process again.
You can remove access here: https://www.spotify.com/de/account/apps/

**Comment by acid-chicken** [comment]
@levrik Hmm, I tried them but the authentication still fails.

**Comment by ShyykoSerhiy** [comment]
Hi @acid-chicken , do you have cookies enabled in firefox?

**Comment by ShyykoSerhiy** [comment]
You can get this error if cookies are disabled.

**Comment by ShyykoSerhiy** [comment]
Theoretically clearing cookies might help too.

**Comment by acid-chicken** [comment]
@ShyykoSerhiy Yes, the cookie is enabled in my Firefox, and I retried after cleaning cookies, but it still fails.

**Comment by Funeoz** [comment]
Found something that can help. I disabled and enabled the extension. Then I started Spotify app and launched a music. After that, I can play/stop music, play previous/next track ... But the Spotify tab in vscode doesn't show anything because the error is still here.
